core/domain/exp_services.py

```diff
@@ -284,6 +287,8 @@ def export_to_zip_file(exploration_id, version=None):
                 feconf.ENTITY_TYPE_EXPLORATION, exploration_id))
         dir_list = fs.listdir('')
         for filepath in dir_list:
+            if filepath.startswith(tuple(DOWNLOAD_RESTRICTED_ASSET_DIRS)):
```

Comment:
Should we allow the user to have an option to download with audio as well? (not sure!)

Reply:
Hm, I'm not sure either. @seanlip would this be useful to have?

Reply:
Nah, I don't think so. Haven't heard of any demand for it.